recipes/biopet-vcfstats/1.1/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Based on OpenJDK recipe in conda-forge
+# https://github.com/conda-forge/openjdk-feedstock/blob/master/recipe/meta.yaml
+# This file was automatically generated by the sbt-bioconda plugin.
+
+     
+test:
```

Comment:
Can you fix the ordering of the keys in the recipe?

Reply:
This is going to take some time, but will work on this. Is there a preferred order? (I guess this  does not matter for the backend at all...) I suppose I should follow the order in https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html?

Reply:
Yes, that would be nice.